### PR TITLE
3.0 Official Hard Launch Release Notes

### PR DESCRIPTION
**Pull request by rr1000**

### CODE REVIEW

File: source/2017-08-16-Platform-3.0-Release-Notes.html.markdown [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+title: The Platform 3.0 Relase Notes
+date: 2017-08-16
+author: ryan
+author_full: Ryan Rich
+tags: compliant cloud, release, 3.0
+---
+
+The latest Platform release, 3.0, is arriving today—August 16th 2017. We’ve spent the better part of the past four months building features that we all believe are principal to our customers' success. The overarching theme is one of self-service, giving customers the ability to do more on their own. For an in-depth look at the release, see the notes below. Additionally, we've written a blog post that goes into even more detail [here](https://datica.com/blog/announcing-self-service-and-free-trials/).
```
Comment (forana) [inline source/2017-08-16-Platform-3.0-Release-Notes.html.markdown]: should add space around the hyphen, and also probably change it to `&mdash;`

File: source/2017-08-16-Platform-3.0-Release-Notes.html.markdown [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+### New Features in 3.0
+**Sensu Checks On Disk Usage Graphing & Alerting:**
+Previous to the 3.0 release our support and developer operations team was required to be proactive about customer disk usage. They would have to manually contact customers on a case by case basis when that customer was approaching their disk usage limits. We realized that in order to make this easier, customers needed a way to see how much disk they were using so they could plan for growth and avoid outages accordingly. In order to eliminate this largely manual process we set up an automatic Sensu check and alert so that the customer can subscribe to this and make changes to their account accordingly.
+
+**Let's Encrypt:**
+One of the largest on-boarding pain points for new customers is the ability to install, manage and update SSL certificates on the platform. Now with Let's Encrypt support, all new environments will come preconfigured with a Let's Encrypt SSL certificate. Additionally, existing customers can change over to Let's Encrypt using the new CLI commands shipping with the Platform 3.0 release.
```
Comment (forana) [inline source/2017-08-16-Platform-3.0-Release-Notes.html.markdown]: add a link to Let's Encrypt somewhere in here

File: source/2017-08-16-Platform-3.0-Release-Notes.html.markdown [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+- Better handling when multiple redeploy calls are sent in quick succession
+- Improvements to in-place redeploy logic
+- No longer an issue with the CLI logs command for a subset of logging containers
+- Better error message when backing up a non-running database
+- Better handling when redeploys fail in certain cases
+- Sending consistent notifications to users when they are added to or removed from groups
+- Many Dashboard fixes with the dashboard rewrite
+
+### Customer Support:
+- In order to tend to your support issue in a timely manner please submit your ticket through the Compliant Cloud dashboard by clicking on the “Contact Support” button located in the footer of the Environment UI. This provides valuable meta data to the support staff which allow them to triage the issue much quicker.
```
Comment (forana) [inline source/2017-08-16-Platform-3.0-Release-Notes.html.markdown]: comma after `timely manner`, and `Compliant Cloud` is in here ;)

File: source/2017-08-16-Platform-3.0-Release-Notes.html.markdown [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+- **Our Support Policies:** Support is provided in English from our offices in Madison, WI.
+- Support hours are Monday through Friday 9:00 a.m. to 5:00 p.m. Central time.
+
+### Additional References:
+For more information on Compliant Cloud and Datica’s offerings, please visit:
```
Comment (forana) [inline source/2017-08-16-Platform-3.0-Release-Notes.html.markdown]: `Compliant Cloud`

File: source/2017-08-16-Platform-3.0-Release-Notes.html.markdown (hunk 1 above)
Comment (rr1000) [inline source/2017-08-16-Platform-3.0-Release-Notes.html.markdown]: Hm. It is an mdash. Looks small on github. Looks fine in browser.

Comment (rr1000) [inline source/2017-08-16-Platform-3.0-Release-Notes.html.markdown]: Also google tells me no spaces. https://www.google.com/search?q=should+an+em+dash+have+spaces+around+it&oq=should+mdash+have+space+around+it%3F&aqs=chrome.1.69i57j0.8695j0j7&sourceid=chrome&ie=UTF-8

File: source/2017-08-16-Platform-3.0-Release-Notes.html.markdown (hunk 2 above)
Comment (rr1000) [inline source/2017-08-16-Platform-3.0-Release-Notes.html.markdown]: Done.

File: source/2017-08-16-Platform-3.0-Release-Notes.html.markdown (hunk 3 above)
Comment (rr1000) [inline source/2017-08-16-Platform-3.0-Release-Notes.html.markdown]: Done.

File: source/2017-08-16-Platform-3.0-Release-Notes.html.markdown (hunk 4 above)
Comment (rr1000) [inline source/2017-08-16-Platform-3.0-Release-Notes.html.markdown]: Done.

File: source/2017-08-16-Platform-3.0-Release-Notes.html.markdown (hunk 1 above)
Comment (forana) [inline source/2017-08-16-Platform-3.0-Release-Notes.html.markdown]: image

**Comment by forana** [comment]
LGTM

**Pull request merged**